README.md

```diff
@@ -6,13 +6,13 @@
 
 This is an open-source suicide / crisis line that runs on PHP and MySQL.
 
-A paid Twilio account is required.  
+A paid Twilio account is required (see Installation:Twilio setup notes below).  
 
 ### How it works
 
 This software creates a help line in the cloud which can be called by anybody that knows the number.  The current version is intended to be used by medium-to-large sized groups and organizations to support themselves, as opposed to a public help line which may have different needs.
 
-With just a web hosting account (that supports php 5.6 and MySQL) and paid Twilio account, you can launch a crisis line in an area code (and possibly number) of your choosing. Twilio costs $1 per month (for the phone number) plus 2 cents per minute (counting both legs of the connection). 
+With just a web hosting account (that supports php 5.6 and MySQL) and paid Twilio account (see Installation:Twilio setup notes below), you can launch a crisis line in an area code (and possibly number) of your choosing. Twilio costs $1 per month (for the phone number) plus 2 cents per minute (counting both legs of the connection). 
```

Comment:
The supported PHP version(s) should be changed to ... ? do we know?

Reply:
7.x - good catch.

Reply:
In looking at the PR code, it *should* run in PHP 5.x and PHP 7.x in that the changes I made were around try/catch code and using PDO rather than MySQL calls. Both functionalities were available in PHP 5. As PHP 5.x installations have some degree of difficulty to install (have to track down code, not part of any available as part of any stock AWS AMIs though available in XAMPP/LAMPP downloads, etc.), how do you feel about the wording:

Been shown to work in PHP 7.x and is believed to run but has not been shown to run in PHP 5.x.